model/cloud/provider.go

```diff
@@ -0,0 +1,7 @@
+package cloud
+
+// Provider is a cloud provider to query LLMs.
```

Comment:
-> `Provider defines a provider to query models such as LLMs.` ? to get rid of the word cloud